interface/procedure_tools/quest/gen_hl7_order.inc.php

```diff
@@ -0,0 +1,590 @@
+<?php
+
+/**
+* Functions to support HL7 order generation.
+*
```

Comment:
@sjpadgett I have no way to test lab Corp or quest. @adunsulag the sandbox for Quest is only for developers. They don't have a test mode. So the test mode I have in there will not work without a developer account.

Reply:
> or quest

If you haven't tested with Quest with this hl7 orders script then why are you putting in here?

Reply:
@sjpadgett you misunderstood. The code has been tested.

Reply:
k @juggernautsei if we can address the LabTransmitEvent review items, I'm fine with bringing this in.  It looks like there isn't a sandbox mode unless we have a developer account with Quest.  Since you've verified this is working on your developer account I'm fine with bringing it in once you address the LabTransmitEvent.

Reply:
If this stays as a singular use event and while I'd prefer this to be a generic event that fires when any  lab order is sent, at least rename the class to reflect it's specific to quest. Otherwise any dev coming behind this PR will think behavior is for any lab send! The naming is deceptive.

Reply:
New class name I believe resolves the issue.

Reply:
I don't understand your thinking that someone would think that all labs used this LabTransmitEvent. Yes, if you just looked only at the name and did nothing else. It could imply that all labs are transported into the ether through this magical conduit. But you can say that about a lot of the naming conventions in the system. That is why a developer needs to investigate. But as you said, some skills are hard to teach. This is not the battle of Little Bighorn. So, I changed the name.